code/datums/wounds/_wounds.dm

```diff
@@ -649,10 +649,20 @@
 	return "[desc]."
 
 /datum/wound/proc/get_scanner_description(mob/user)
-	return "Type: [name]\nSeverity: [severity_text(simple = FALSE)]\nDescription: [desc]\nRecommended Treatment: [treat_text]"
+	var/wound_status_info = get_wound_status_info()
+	var/wound_status_notes = (wound_status_info ? "\nWound status: [wound_status_info]" : "")
```

Comment:
this would fail to compile since we're using a non-existant variable

its more efficient to store the data in a variable instead of calling it twice in a ternary